tools/project_manager/show_all_good_word_suggestions.php

```diff
@@ -7,7 +7,8 @@
 include_once($relPath.'Stopwatch.inc');
 include_once($relPath.'misc.inc'); // array_get(), get_integer_param(), surround_and_join()
 include_once('./post_files.inc');
-include_once("./word_freq_table.inc");
+include_once("./word_freq_table.inc"); // echo_cutoff_text()
+include_once($relPath.'page_controls.inc');
```

Comment:
Please move this up above line 9 so that all of the `$relPath` includes are grouped together

Reply:
Done this.